specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/iotDefenderSettings.json

```diff
@@ -0,0 +1,445 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2020-08-06-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/iotDefenderSettings": {
+      "get": {
+        "x-ms-examples": {
+          "List IoT Defender settings": {
+            "$ref": "./examples/IotDefenderSettings/List.json"
+          }
+        },
+        "tags": [
+          "IoT Defender Settings"
+        ],
+        "description": "List IoT Defender Settings",
+        "operationId": "IotDefenderSettings_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IotDefenderSettingsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/iotDefenderSettings/default": {
```

Comment:
>/subscriptions/{subscriptionId}/ [](start = 5, length = 32)

Is it  a Proxy resource? Tracked resources are only modeled under a RG.

Reply:
Yes, this would be a proxy resource.